scripts/local-dev.sh

```diff
@@ -11,12 +11,13 @@ NC='\033[0m' # No Color
 
 # Configuration
 PROMETHEUS_NAMESPACE="openshift-monitoring"
-LLM_NAMESPACE="${LLM_NAMESPACE:-m3}"
+LLM_NAMESPACE="${LLM_NAMESPACE:-test1}"
```

Comment:
I don't think we should change LLM_NAMESPACE in the script. It should default to whatever value and we should override it from cli.

Reply:
I agree, but if run with default value for namespace we don't have `m3` namespace with deployment installed, so it cause issues 
do you want me to keep this empty ?

Reply:
I think, we should use `dev` namespace as we agreed earlier this week. We will need to create and deploy to that namespace first though.

@tsisodia10 any suggestions?

Reply:
let's cover this next PRs, once deployed into `dev`